packages/react/src/ActionMenu/ActionMenu.docs.json

```
@@ -10,7 +10,7 @@
       "type": "React.ReactElement[]",
       "defaultValue": "",
       "required": true,
-      "description": "Recommended: `ActionMenu.Button` or `ActionMenu.Anchor` with `ActionMenu.Overlay`"
+      "description": "Recommended: `ActionMenu.Button`, or `ActionMenu.Anchor`, with `ActionMenu.Overlay`"
```

```suggestion
      "description": "Recommended: `ActionMenu.Button` or `ActionMenu.Anchor` with `ActionMenu.Overlay`"
```

Should we revert it to its original state? Do we need a comma before or? 🤔

---

good catch :+1: